tools/ci_build/github/azure-pipelines/templates/web-ci.yml

```diff
@@ -19,10 +19,19 @@ parameters:
   displayName: 'Build static library'
   type: boolean
   default: false
+- name: BuildJsep
```

Comment:
I suggest adding a part to https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/github/azure-pipelines/post-merge-jobs.yml to verify the code still works fine when BuildJsep=false.